src/HDF5FileLayout.cpp

```diff
@@ -67,7 +67,7 @@ HDF5FileLayout::get_record_number_string(uint64_t record_number, // NOLINT(build
   int width = m_conf_params.digits_for_record_number;
 
   if (record_number >= m_powers_ten[m_conf_params.digits_for_record_number]) {
-    ers::warning(FileLayoutNotEnoughDigitsForPath(ERS_HERE, record_number, m_conf_params.digits_for_record_number));
+    //ers::warning(FileLayoutNotEnoughDigitsForPath(ERS_HERE, record_number, m_conf_params.digits_for_record_number));
```

Comment:
I would vote for either removing this line entirely, or perhaps making it a TLOG_DEBUG so we still have visibility when the minimum is insufficient.

Reply:
As Eric suggested, I removed the lines entirely.  (And, in fact, I took the opportunity to remove a few more lines of unnecessary code.)